scripts/auth-token-manager.js

```diff
@@ -0,0 +1,71 @@
+/*
+ * Copyright (c) 2019, salesforce.com, inc.
+ * All rights reserved.
+ * SPDX-License-Identifier: MIT
+ * For full license text, see the LICENSE file in the repo root or https://opensource.org/licenses/MIT
+ */
+
+// Borrowed from https://github.com/facebook/jest
+
+/**
+ * Copyright (c) 2014-present, Facebook, Inc. All rights reserved.
+ *
+ * This source code is licensed under the MIT license found in the
+ * LICENSE file in the root directory of this source tree.
+ */
+
+'use strict';
+
+const jwt = require('jsonwebtoken');
+const readline = require("readline");
+
+const rl = readline.createInterface({
+     input: process.stdin,
+     output: process.stdout
+});
+
+function generateNewToken() {
+    rl.question("Who are you generating this token for? ", function (user) {
+        rl.question("How long should this token last for? (e.g. '1 year', '2 days', '24 hours', '5 minutes') ", function (validFor) {
+            const token = jwt.sign({ user }, process.env.TOKEN_SECRET, { expiresIn: validFor });
+            const expiration = new Date(jwt.decode(token).exp * 1000);
+            console.log(`Token generated for '${user}' expires on ${expiration}:\n${token}`);
+            rl.close();
+        });
+    });
+}
```

Comment:
@dbajric For [consistency](https://github.com/salesforce/best/pull/294/files#diff-d1fac42249ae35f6a31e49720a9903a2c399fabf63d35fa2038cef26b518b5baR47), maybe switch to using arrow functions here and everywhere else?

```suggestion
const generateNewToken = () => {
    rl.question("Who are you generating this token for? ", (user) => {
        rl.question("How long should this token last for? (e.g. '1 year', '2 days', '24 hours', '5 minutes') ", (validFor) => {
            const token = jwt.sign({ user }, process.env.TOKEN_SECRET, { expiresIn: validFor });
            const expiration = new Date(jwt.decode(token).exp * 1000);
            console.log(`Token generated for '${user}' expires on ${expiration}:\n${token}`);
            rl.close();
        });
    });
};
```

Reply:
Converted to arrow functions.